doc/include/bitmask_usage.rst

```diff
@@ -0,0 +1,110 @@
+
+Bitmasks allow you to define a set of bit values signified by strings,
+and then toggle and interpret bits held by a `numpy.ndarray`_.  For
+example, say you're processing an image and you want to set up a set of
+bits that indicate that the pixel is part of a bad-pixel mask, has a
+cosmic ray, or is saturated.  You can define the following::
+
+    from collections import OrderedDict
```

Comment:
You might drop the `OrderedDict` part.  We are fully on Python >= 3.7

Reply:
Dropped and added some explanation.